Packages/com.unity.inputsystem/InputSystem/Devices/Gamepad.cs

```diff
@@ -326,64 +326,6 @@ public enum GamepadButton
         /// The right trigger button on a gamepad.
         /// </summary>
         RightTrigger = 33,
-
```

Comment:
We cannot remove these from this enum without requiring a major version revision since it would break public API for existing customers (and according to semver), hence this is not a viable way forward.